test/test/console-log.tap

```diff
@@ -18,9 +18,10 @@ TAP version 13
         ok 4 - nested ok second
         1..4
     ok 2 - second ___/# time=[0-9.]+(ms)?/~~~
-ok 1 - nesting ___/# time=[0-9.]+(ms)?/~~~
 
 >>>> after second child
+ok 1 - nesting ___/# time=[0-9.]+(ms)?/~~~
+
```

Comment:
Even if they aren't intentionally relying on it, having their tests suddenly start failing without changing their own code would likely be seen as a breaking change. Given the number of major version bumps in the last 12 months I think most tap users would expect one for this change.